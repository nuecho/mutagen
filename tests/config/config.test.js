test("[mutagen config --help] should print config usage", () => {
  const {code, output} = mutagen(`config --help`);

  expect(output).toMatchSnapshot("config usage");
  expect(code).toBe(0);
});
