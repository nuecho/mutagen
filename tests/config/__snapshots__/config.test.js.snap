// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen config --help] should print config usage: config usage 1`] = `
Object {
  "stderr": "",
  "stdout": "Usage: config [-?]
Genesys Config Server tool
  -?, -h, --help              Show this help message.
Commands:
  export    Export configuration objects.
  import    [INCUBATION] Import configuration objects.
  validate  Validate the configuration objects against the Configuration Server.
",
}
`;

exports[`[mutagen config export --trust-insecure-certificate] should allow connecting tls even without ca certificate: export without exception 1`] = `""`;

exports[`[mutagen config export] should not allow connecting tls if ca certificate is missing: error while connecting 1`] = `
"Error while connecting to Config Server [configserver:2022].
"
`;
